src/custom/components/swap/SwapModalHeader/SwapModalHeaderMod.tsx

```diff
@@ -27,6 +27,13 @@ import TradeGp from 'state/swap/TradeGp'
 import { INPUT_OUTPUT_EXPLANATION } from 'constants/index'
 import { computeSlippageAdjustedAmounts } from 'utils/prices'
 import { Field } from 'state/swap/actions'
+import { darken } from 'polished'
+
+// MOD
+const LightCard = styled(LightCardUni)`
```

Comment:
Example of mod changes.

I would expect these to be in the index, and the parent component to take this custom component as a prop.